appengine/standard_python3/building-an-app/building-an-app-3/templates/index.html

```diff
@@ -18,33 +18,23 @@
 <html>
 <head>
   <title>Datastore and Firebase Auth Example</title>
-
-  <!-- See https://github.com/firebase/firebaseui-web. -->
-  <!-- [START gae_python38_auth_init_firebase] -->
```

Comment:
Sorry, really dumb question: I gather from snippets-bot that these are region tags, and I guess they're used to anchor in on a subset of a larger file, so a particularly interesting function can be emphasized in documentation? Or maybe they're a maintenance thing so imports can be bulk updated to new version numbers? I can't tell what these are for to know if deleting them matters.

It would be easy enough to restore them but they don't make for very pleasant reading, so I wanted to check first.

Reply:
Hi Samuel. The region tags are identifiers used by our documentation to embed code snippets into the code sample widget. For example, here's a [tutorial page](https://cloud.google.com/appengine/docs/standard/python3/building-app/authenticating-users)  that includes code snippets. So, they are required, or otherwise we'll lose the references in the documentation. Thanks for your contribution!

Reply:
Thanks Karl! That makes sense. If I'm reading the snippet-bot finding right it looks like one of these tags is actively being used 

* gae_python38_auth_include_firebaseui

and another 7 are not:

* gae_python38_auth_init_firebase
* gae_python3_auth_init_firebase
* gae_python3_auth_include_firebaseui
* gae_python38_auth_init_firebase
* gae_python3_auth_init_firebase
* gae_python38_auth_include_firebaseui
* gae_python3_auth_include_firebaseui

I'm happy to add the first one back in - is it okay if I drop the rest? Normally I'm all for preserving anything somebody went to the effort to add, but I feel like it hampers readability a little on such a short file - nearly half of the lines are comments. Perhaps they are all being used and the bot isn't doing an exhaustive search and this is a silly question :)

Reply:
Preserved gae_python38_auth_include_firebaseui and gae_python38_auth_firebase_html per discussion elsewhere.